modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/filter/expression/PredicateExpressionVisitorImpl.java

```diff
@@ -437,7 +440,41 @@ private EntityModel _getObjectDefinitionEntityModel(
 	}
 
 	private Predicate _getPredicateForRelationships(
-			BinaryExpression.Operation operation, Object left, Object right)
+		Object left,
+		UnsafeBiFunction<String, Long, Predicate, Exception> unsafeBiFunction) {
```

Comment:
Can you give this a more semantic name? I have no clue what it is just by reading it.

Reply:
I'm referring to `unsafeBifunction`.

Reply:
Done!
I leaved it as `unsafeBiFunction` because most of the examples in the portal leave it as it is. I assumed that should be some kind of convention in Liferay 🤷‍♂️